pom.xml

```
@@ -53,16 +53,16 @@
         <version.org.eclipse.microprofile.lra>2.0</version.org.eclipse.microprofile.lra>
         <version.io.micrometer>1.12.4</version.io.micrometer>
         <version.org.jboss.arquillian>1.7.2.Final</version.org.jboss.arquillian>
-        <version.org.jboss.wildfly.dist>32.0.1.Final</version.org.jboss.wildfly.dist>
+        <version.org.jboss.wildfly.dist>34.0.0.Beta1</version.org.jboss.wildfly.dist>
```

Final version is live as of today

```suggestion
        <version.org.jboss.wildfly.dist>34.0.0.Final</version.org.jboss.wildfly.dist>
```

---

Thanks!